docs/content.zh/docs/dev/table/sql/truncate.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "TRUNCATE 语句"
+weight: 8
+type: docs
+aliases:
+- /zh/dev/table/sql/truncate.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<a name="truncate-statements"></a>
+
+# TRUNCATE 语句
+
+TRUNCATE 语句用于删除表中的全部数据，但不会删除表本身。
+<span class="label label-danger">注意</span> 目前, `TRUNCATE` 语句仅支持批模式, 并且要求目标表实现了 {{< gh_link file="flink-table/flink-table-common/src/main/java/org/apache/flink/table/connector/sink/abilities/SupportsTruncate.java" name="SupportsTruncate" >}} 接口。
+如果在一个没有实现该接口的表上执行 `TRUNCATE`，则会抛异常。目前 Flink 内置的连接器还没有实现该接口。
+
+<a name="run-a-truncate-statement"></a>
+
+## 执行 TRUNCATE 语句
+
+{{< tabs "truncate" >}}
+{{< tab "Java" >}}
```

Comment:
Why chinese doc miss `Scala/Python` in tab title?
<img width="998" alt="image" src="https://github.com/apache/flink/assets/20389154/32b2722a-14c3-41f4-8b72-94897baf7fa0">